### PR TITLE
Add Starting Soon events

### LINKED ISSUE #39: players entered after tournament start time are no longer able to add a decklist from player cp
Reported on Discord by bakert#2193

### PR DESCRIPTION
**Pull request by stash86**
Fixes #39

### CODE REVIEW

File: gatherling/models/Event.php [hunk 1]
```diff
@@ -1027,6 +1027,26 @@ public static function getNextPreRegister($num = 20)
         return $events;
     }
 
+    public static function getStartingSoon($playername)
+    {
+        $db = Database::getConnection();
+        $stmt = $db->prepare('SELECT e.name FROM events e, entries n WHERE n.event_id = e.id AND n.player = ? AND prereg_allowed = 1 AND active = 0 AND finalized = 0 AND DATE_SUB(start, INTERVAL 0 MINUTE) < NOW() ORDER BY start');
```
Comment (silasary) [inline gatherling/models/Event.php]: Why only prereg events?

Comment (stash86) [reply gatherling/models/Event.php]: Ah yeah. Forgot that some events might not allow prereg. Revising it now

Comment (stash86) [reply gatherling/models/Event.php]: On another note, the Your Upcoming Events will only show prereg events as well. Do we want to change it also? @silasary

**Review by silasary — APPROVED** [review]
Well.... it works.  For better or for worse.
image

**Pull request merged**